### PR TITLE
fix(CMake.MSVC.Build): provide option for portable installation

### PR DESCRIPTION
**Pull request by stef145g**
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/1306)

### CODE REVIEW

**Comment by slodki** [comment]
But now we created 2 packages from one build. Hard-coding this file location in build configuration will require to compile MMEX twice.
If this file is not required in one package this should be detected and managed at install time not configuration.

What are the consequences of adding this file to installer (not portable package)?

File: CMakeLists.txt [hunk 1]
```diff
@@ -291,8 +293,8 @@ IF(MSVC_IDE)
     MESSAGE(STATUS "Tuning for VisualStudio IDE:")
 
     IF(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-        SET(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/ide"
-            CACHE PATH "Install directory used by INSTALL target" FORCE)
+        SET(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/moneymanagerex"
+            CACHE PATH "Install directory used by INSTALL target\nChange 'moneymanagerex' to 'src/Release' or 'src/Debug' for use in IDE" FORCE)
```
Comment (slodki) [inline CMakeLists.txt]: Don't do it. It's fixed already in my `update-cmake2` branch.
Now it's as easy as:
1. File -> Open directory...
2. CMake -> Install
3. CMake -> Debug

I'm working on the BUILD.md (testing on my VS fresh install) to prepare PR for this.

File: CMakeLists.txt [hunk 2]
```diff
@@ -425,44 +427,45 @@ ENDIF()
 MESSAGE(STATUS "")
 MESSAGE(STATUS "MMEX configuration summary")
 MESSAGE(STATUS "==========================")
-MESSAGE(STATUS "Version        : ${MMEX_VERSION_WITH_UNSTABLE}")
+MESSAGE(STATUS "Version         : ${MMEX_VERSION_WITH_UNSTABLE}")
```
Comment (slodki) [inline CMakeLists.txt]: I've changed many things here (even fixing upstream bugs in CMake)...

**Comment by slodki** [comment]
I've studied CPack documentation and it support multiple install scenarios for different packages type. We need use [CPackComponent](https://cmake.org/cmake/help/latest/module/CPackComponent.html) for this proposes.

**Comment by slodki** [comment]
Some [examples](https://cmake.org/Wiki/CMake:Component_Install_With_CPack) of using CPackComponent.

**Comment by stef145g** [comment]
Since a similar change is in progress, I will close this PR

**Pull request closed without merging**